cmd/lambda/publish_lease_events/main.go

```diff
@@ -94,38 +94,32 @@ func handleRecord(input *handleRecordInput) error {
 		prevLeaseStatus := prevLeaseStatusAttr.String()
 
 		// Grab the new lease status
-		nextLeaseStatusAttr, ok := record.Change.NewImage["LeaseStatus"]
+		newLeaseStatusAttr, ok := record.Change.NewImage["LeaseStatus"]
```

Comment:
just a var name change -- I got confused, thinking that the `nextLeaseStatus` had not yet been applied.